### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by johnoneill46**
Just minor code cleanups like adding missing semi-colons, getting rid of redundant things etc

### CODE REVIEW

File: common/steps/progress-bar/ProgressBar.step.js [hunk 1]
```diff
@@ -12,10 +12,6 @@ class ProgressBar extends Interstitial {
     return this.req.session.case.data;
   }
 
-  get caseId() {
-    return this.req.session.case.caseId;
-  }
-
```
Comment (johnoneill46) [inline common/steps/progress-bar/ProgressBar.step.js]: Removed this as it wasn't being used

File: mocks/services/case-orchestration/retrieve-aos-case/mock-case-resp.json [hunk 2]
```diff
@@ -189,9 +189,6 @@
     "paymentPhoneNumber": "1234567332",
     "divorceCostsClaim": "Yes",
     "divorceIsNamed": "Yes",
-    "claimsCostsFrom": [
-      "respondent"
-    ],
```
Comment (johnoneill46) [inline mocks/services/case-orchestration/retrieve-aos-case/mock-case-resp.json]: Removed this as it's a duplicate key - there were two "claimsCostsFrom" in the json

File: mocks/services/case-orchestration/retrieve-case/mock-case.json [hunk 3]
```diff
@@ -189,9 +189,6 @@
     "paymentPhoneNumber": "1234567332",
     "divorceCostsClaim": "Yes",
     "divorceIsNamed": "Yes",
-    "claimsCostsFrom": [
-      "respondent"
-    ],
```
Comment (johnoneill46) [inline mocks/services/case-orchestration/retrieve-case/mock-case.json]: Removed this as it's a duplicate key - there were two "claimsCostsFrom" in the json

File: mocks/services/case-orchestration/submit-da/__/mock-case.json [hunk 4]
```diff
@@ -187,9 +187,6 @@
       "paymentPhoneNumber": "1234567332",
       "divorceCostsClaim": "Yes",
       "divorceIsNamed": "Yes",
-      "claimsCostsFrom": [
-        "respondent"
-      ],
```
Comment (johnoneill46) [inline mocks/services/case-orchestration/submit-da/__/mock-case.json]: Removed this as it's a duplicate key - there were two "claimsCostsFrom" in the json

File: mocks/steps/modify-session/ModifySession.html [hunk 5]
```diff
@@ -13,11 +13,10 @@
   <script src="/session/assets/jsoneditor/jsoneditor.min.js"></script>
   <style>
     .form-group{
-      margin:0;
-      margin-bottom: 30px;
+      margin: 0 0 30px;
     }
     .form-group .heading-medium{
-      margin-top: 0px;
```
Comment (johnoneill46) [inline mocks/steps/modify-session/ModifySession.html]: No need for px (or any unit) in HTML when the value is 0

File: mocks/steps/modify-session/ModifySession.html [hunk 6]
```diff
@@ -13,11 +13,10 @@
   <script src="/session/assets/jsoneditor/jsoneditor.min.js"></script>
   <style>
     .form-group{
-      margin:0;
-      margin-bottom: 30px;
```
Comment (johnoneill46) [inline mocks/steps/modify-session/ModifySession.html]: Changed as its a cleaner way of writing the same thing

File: steps/respondent/progress-bar/RespondentProgressBar.step.js [hunk 7]
```diff
@@ -45,14 +45,12 @@ class RespondentProgressBar extends ProgressBar {
...
-        pageContent = contentMap.daRequested;
-        break;
-      default:
-        pageContent = contentMap.divorceGranted;
+    if (this.currentCaseState.toLowerCase() === caseStates.daRequested) {
+      pageContent = contentMap.daRequested;
+    } else {
+      pageContent = contentMap.divorceGranted;
     }
-    /* eslint-enable indent */
```
Comment (johnoneill46) [inline steps/respondent/progress-bar/RespondentProgressBar.step.js]: This was a redundant switch statement as there was only a single option anyway

File: assets/js/showHideContent.js [hunk 8]
```diff
@@ -1,34 +1,34 @@
...
+      var $control = jQuery(this);
+      var $content = getToggledContent($control);
 
       // Set aria-controls and defaults
       if ($content.length) {
-        $control.attr('aria-controls', $content.attr('id'))
-        $control.attr('aria-expanded', 'false')
+        $control.attr('aria-controls', $content.attr('id'));
+        $control.attr('aria-expanded', 'false');
         $content.attr('aria-hidden', 'true')
```
Comment (Dizrupto) [inline assets/js/showHideContent.js]: Should this line have the semi-colon as well? :)

File: assets/js/showHideContent.js [hunk 9]
```diff
@@ -141,25 +141,25 @@
...
-      $container = $container || jQuery(document.body)
+      $container = $container || jQuery(document.body);
       $container.off('.' + selectors.namespace)
     }
   }
 
   ShowHideContent.prototype.init = function ($container) {
-    this.showHideRadioToggledContent($container)
+    this.showHideRadioToggledContent($container);
     this.showHideCheckboxToggledContent($container)
```
Comment (Dizrupto) [inline assets/js/showHideContent.js]: Also here, should there be a semi-colon on this line? :)

File: assets/js/showHideContent.js (hunk 8 above)
Comment (iichr) [inline assets/js/showHideContent.js]: No impact on anything, just a style observation - I suppose we don't have a js style agreed on for this repo, few other places not consistent in (not) using `;`

File: .github/CONTRIBUTING.md [hunk 10]
```diff
@@ -66,4 +66,4 @@ Open a new pull request with a single commit including the above changes.
 
 [Here is an example for v6.1.0](https://github.com/alphagov/govuk_prototype_kit/commit/53e36d79a994ce3649b53f4008370cd75068c27c).
 
-Once merged into master a new version will be built.
\ No newline at end of file
+Once merged into master a new version will be built.
```
Comment (yarpo) [inline .github/CONTRIBUTING.md]: what was changed here :) ?

Comment (johnoneill46) [reply .github/CONTRIBUTING.md]: Nothing, not sure why Github picked up on it 🤷‍♂

File: mocks/steps/modify-session/ModifySession.html (hunk 5 above)
Comment (yarpo) [reply mocks/steps/modify-session/ModifySession.html]: Have you double checked that 0 px is the same as 0 in? ;)

**Pull request merged**